provider/provider_yaml_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2016-2023, Pulumi Corporation.  All rights reserved.
+//
+// Experimental provider upgrade tests.
+//
+// The objective of these tests is to make sure the provider release candidate will not generate any
+// Update or Replace plans for Pulumi stacks deployed on a previous baseline version of the
+// provider.
+//
+// Note on build flags above --^ these tests currently uses YAML only but our build workflow matrix
```

Comment:
This file doesn't have build tags either.

Reply:
Good catch, fixing up.